docs/guide/learning/books.md

```diff
@@ -175,3 +175,19 @@ Readers will then use Laravel to set up a web service and integrate the front en
 </useful-links-section>
 </useful-links>
 
+### Pro Vue.js 2
+**Author:** Adam Freeman
+**Publisher:** Apress
```

Comment:
We can drop the publisher field.

Reply:
> We can drop the publisher field.

Isn't it useful what publishing house (if any) sells the book? For example I have subscription to access all Packt books and can read all their books for constant monthly fee.

But if you intentionally omit publisher information, I'll remove it.

Reply:
Alright, leave it then. We need to add the rest in that case.